coral/src/app/layout/main-navigation/MainNavigation.test.tsx

```diff
@@ -354,12 +354,6 @@ describe("MainNavigation.tsx", () => {
         expect(button).toBeEnabled();
       });
     });
-
```

Comment:
Why would we delete this? 🤔

Reply:
This tests seems to assert the exact same thing in a more thorough and reliable way: https://github.com/Aiven-Open/klaw/blob/41779d9e909ba38f8bce4c290d0c66ffa3ff8780/coral/src/app/layout/main-navigation/MainNavigation.test.tsx#L338-L357

Additionally, because this test counts button, it also counts the button in the feedback CTA, so its assertion `renders all submenu buttons` is wrong. And changing the test to assert " renders all buttons" seems superfluous.

Reply:
💡  Makes sense!